<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account</class>
 <widget class="QDialog" name="Account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Аккаунт</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #2c3e50;
    }
    QLabel {
        color: #ecf0f1;
        font-size: 14pt;
        font-weight: bold;
    }
    QLineEdit {
        background-color: #ecf0f1;
        color: #2c3e50;
        font-size: 12pt;
        padding: 5px;
        border-radius: 5px;
    }
    QComboBox {
        background-color: #ecf0f1;
        color: #2c3e50;
        font-size: 12pt;
        padding: 5px;
        border-radius: 5px;
    }
    QPushButton {
        background-color: #3498db;
        color: #ecf0f1;
        font-size: 14pt;
        font-weight: bold;
        border-radius: 10px;
        padding: 10px;
        min-width: 200px;
        margin: 10px;
    }
    QPushButton:hover {
        background-color: #2980b9;
    }
    QPushButton:pressed {
        background-color: #1abc9c;
    }
   </string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>300</width>
     <height>500</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Введите данные аккаунта</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameEdit">
      <property name="placeholderText">
       <string>Имя аккаунта</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordEdit">
      <property name="placeholderText">
       <string>Пароль</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="roleComboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Ученик</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Преподаватель</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="confirmButton">
      <property name="text">
       <string>Подтвердить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
